tasks.rb

```diff
@@ -3924,6 +3943,32 @@ def set_hpxml_ceiling_fans(hpxml_file, hpxml)
   end
 end
 
+def set_hpxml_pools(hpxml_file, hpxml)
+  if ['base-misc-large-uncommon-loads.xml'].include? hpxml_file
+    hpxml.pools.add(id: 'Pool',
+                    heater_type: HPXML::HeaterTypeElectric,
+                    heater_kwh_per_year: 100,
+                    pump_kwh_per_year: 50)
+    hpxml.pools.add(id: 'Pool2',
+                    heater_type: HPXML::HeaterTypeGas,
+                    heater_therm_per_year: 100,
+                    pump_kwh_per_year: 50)
```

Comment:
I think it'd make more sense to create a base-misc-large-uncommon-loads2.xml file, based off base-misc-large-uncommon-loads.xml, where you have one pool/hot tub in each but change the fuel type and energy use.

Reply:
Didn't read this before I responded to the last comment. Good idea.